src/data/finset/lattice.lean

```diff
@@ -37,6 +37,10 @@ fold_cons h
 @[simp] lemma sup_insert [decidable_eq β] {b : β} : (insert b s : finset β).sup f = f b ⊔ s.sup f :=
 fold_insert_idem
 
+lemma sup_image [decidable_eq β] (s : finset γ) (f : γ → β) (g : β → α):
```

Comment:
Great. It's consistent with `sup_image` above. Mathlib style seems to suggest this name is the better one, but there are exceptions everywhere.

Reply:
I presume you mean `sup_insert`? I think since `sup` takes two arguments I'd expect a prefix to be the first argument and a suffix to be the second, like `sup_const` and `map_sup`.